package.json

```
@@ -13,22 +13,22 @@
     "./use-at-your-own-risk": "./lib/unsupported-api.js"
   },
   "scripts": {
+    "build:docs:update-links": "node tools/fetch-docs-links.js",
+    "build:release": "node Makefile.js generateRelease",
+    "build:release:alpha": "node Makefile.js generatePrerelease -- alpha",
+    "build:release:beta": "node Makefile.js generatePrerelease -- beta",
+    "build:release:publish": "node Makefile.js publishRelease",
+    "build:release:rc": "node Makefile.js generatePrerelease -- rc",
```

```suggestion
    "release:generate:latest": "node Makefile.js generateRelease",
    "release:generate:alpha": "node Makefile.js generatePrerelease -- alpha",
    "release:generate:beta": "node Makefile.js generatePrerelease -- beta",
    "release:publish": "node Makefile.js publishRelease",
    "release:generate:rc": "node Makefile.js generatePrerelease -- rc",
```

Thoughts about these names? I think we should update the standard to separate release scripts from build scripts.

---

What is the distinction between "build" and "release"?

---

In my opinion,

"build" scripts should only create/update files locally and should be safe to run anytime by anyone.

"release" script can publish, commit, tag, push, etc. They should be run only during the release process.

---

Makes sense to me 👍🏻

---

Sounds good to me. Let's be sure to update the conventions to reflect this.

---

Okay I’ll add those changes to the PR this weekend. Thanks for the feedback!